source/api/roleToBaseclass.service.ts

```diff
@@ -66,10 +66,10 @@ export class RoleToBaseclassService {
         ).pipe(map((o: any) => FlexiCoreDecycle.retrocycle(o)));
     }
 
-    public getRoleToBaseclasses(body?: RoleToBaseclassFiltering, authenticationKey?: string, extraHttpRequestParams?: any, observe?: 'body', reportProgress?: boolean): Observable<PaginationResponse<RoleToBaseclass>>;
-    public getRoleToBaseclasses(body?: RoleToBaseclassFiltering, authenticationKey?: string, extraHttpRequestParams?: any, observe?: 'response', reportProgress?: boolean): Observable<PaginationResponse<HttpResponse<RoleToBaseclass>>>;
-    public getRoleToBaseclasses(body?: RoleToBaseclassFiltering, authenticationKey?: string, extraHttpRequestParams?: any, observe?: 'events', reportProgress?: boolean): Observable<HttpEvent<PaginationResponse<RoleToBaseclass>>>;
-    public getRoleToBaseclasses(body?: RoleToBaseclassFiltering, authenticationKey?: string, extraHttpRequestParams?: any, observe: any = 'body', reportProgress: boolean = false): Observable<any> {
+    public getRoleToBaseclasses(body?: RoleToBaseclassFilter, authenticationKey?: string, extraHttpRequestParams?: any, observe?: 'body', reportProgress?: boolean): Observable<PaginationResponse<RoleToBaseclass>>;
```

Comment:
use same names for apis , this api is called getAll not getRoleToBaseclass,
additionally the /api should be part of the basepath and should not be automatically added, so it should be :
${this.basePath}/roleToBaseclass/getAll

this is because the api part is configurable serverside so its not mandatory that it will be present or that it will be called api